src/main/java/com/aws/greengrass/lifecyclemanager/Kernel.java

```diff
@@ -536,7 +537,7 @@ private Class<?> locateExternalPlugin(String name, Topics serviceRootTopics) thr
         if (storedDigest == null || storedDigest.getOnce() == null) {
             logger.atError("plugin-load-error").kv(GreengrassService.SERVICE_NAME_KEY, name)
                     .log("Local external plugin is not supported by this greengrass version");
-            throw new ServiceLoadException("Custom plugins is not supported by this greengrass version");
+            throw new CustomPluginNotSupportedException("Custom plugins is not supported by this greengrass version");
```

Comment:
let's fix this error message. `Locally deployed plugin components are not supported. Plugins must be deployed via a cloud-based deployment.`

Reply:
updating